docs/Reference/Errors.md

```diff
@@ -290,6 +290,23 @@ The logger accepts either a `'stream'` or a `'file'` as the destination.
 The logger should have all these methods: `'info'`, `'error'`,
 `'debug'`, `'fatal'`, `'warn'`, `'trace'`, `'child'`.
 
+#### FST_ERR_LOG_INVALID_LOGGER_INSTANCE
+<a id="FST_ERR_LOG_INVALID_LOGGER_INSTANCE"></a>
+
+The loggerInstance only accepts a logger instance, not a configuration object. 
```

Comment:
https://fastify.dev/docs/latest/Reference/Logging/

Seems still oudated? Received this error with old `logger: {info...}`

Reply:
First, this is over a year old. Open an issue if you have encountered a problem. Second, you have not provided enough information to understand what you are asking. Please provide details in your issue.